apple/Sources/FerrostarMapLibreUI/Views/DynamicallyOrientingNavigationView.swift

```diff
@@ -73,6 +74,22 @@ public struct DynamicallyOrientingNavigationView: View, CustomizableNavigatingIn
                     orientation: orientation,
                     geometry: geometry
                 ))
+                .onChange(of: geometry.safeAreaInsets) { newValue in
+                    overlaySafeAreaInsets = newValue.collapsedChildInsets(minimumInset:  16)
+                }
+                // For debug visualizing of overlaySafeAreaInsets
+//                .overlay(
+//                    ZStack {
+//                        // MapView
+//                        Rectangle().fill(.red.opacity(0.5)).ignoresSafeArea()
+//
+//                        // MapView's safeArea
+//                        Rectangle().fill(.yellow.opacity(0.5))
+//
+//                        // Overlay safeArea
+//                        Rectangle().fill(.blue.opacity(0.5)).padding(overlaySafeAreaInsets)
+//                    }
+//                )
```

Comment:
I can remove this if you like. It might help you review to see what it looks like though - see the code/comments for the color key:

<img width="300" alt="Screenshot 2024-09-18 at 20 09 00" src="https://github.com/user-attachments/assets/476c10fb-a8b0-4880-9683-0c8e65e9561e">
<img width="400" alt="Screenshot 2024-09-18 at 20 09 03" src="https://github.com/user-attachments/assets/f8c2727e-d69a-470e-a35e-9fa4cfe5ceb7">
<img width="300" alt="Screenshot 2024-09-18 at 20 09 18" src="https://github.com/user-attachments/assets/77d8ba3d-e275-4476-a212-1ac0b42cb627">
<img width="400" alt="Screenshot 2024-09-18 at 20 09 22" src="https://github.com/user-attachments/assets/1a2748e4-a5a9-4005-ab16-b21756990255">